src/appkit.cs

```diff
@@ -4795,7 +4795,7 @@ partial interface NSControl {
 		[Export ("font")]
 		NSFont Font { get; set; }
 
-		[Export ("formatter", ArgumentSemantic.Retain)]
+		[Export ("formatter", ArgumentSemantic.Retain), NullAllowed]
 		NSObject Formatter { get; set; }
```

Comment:
According to the latest doc (did not check headers) it's an `NSFormatter` or a subclass (`__kindof`) so, to avoid forgetting it, maybe
```csharp
#if XAMCORE_4_0
NSFormatter Formatter { get; set; }
#else
NSObject Formatter { get; set; }
#endif
```